src/sentry/services/hybrid_cloud/access/service.py

```diff
@@ -38,14 +38,6 @@ def can_override_sso_as_owner(
         """
         pass
 
-    @abc.abstractmethod
-    def get_all_org_roles(self, member_id: int, organization_id: int) -> list[str]:
-        pass
-
-    @abc.abstractmethod
-    def get_top_dog_team_member_ids(self, organization_id: int) -> list[int]:
-        pass
```

Comment:
Removing RPC methods requires two deploys.

1. First remove all callers of an RPC method, and deploy that.
2. Once your callsite changes have landed you can remove the RPC method from the service and impl modules.

Doing both changes in one deployment will cause a burst of failures until all regions have been deployed which we should avoid.

Reply:
i see, i'll revert this one and separate them out 👍